src/main/kotlin/com/hibob/academy/dao/models.kt

```diff
@@ -1,13 +1,17 @@
 package com.hibob.academy.dao
 
-import java.util.*
+import java.sql.Date
 
 data class Example(val id: Long, val companyId: Long, val data: String)
 
 data class OwnerData(val id: Long, val name: String, val companyId: Long, val employeeId: String)
 
+data class Owner(val name: String, val companyId: Long, val employeeId: String)
```

Comment:
I understand know, please rename this data class to something we can understand like OwnerCreationRequesr